spikeinterface/sorters/si_based_sorters/spyking_circus2.py

```diff
@@ -43,19 +44,17 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
 
         assert HAVE_HDBSCAN, 'spykingcircus2 needs hdbscan to be installed'
     
-        params['job_kwargs']['verbose'] = verbose
-        params['job_kwargs']['progress_bar'] = verbose
-
-        if "n_jobs" in params["job_kwargs"]:
-            if params["job_kwargs"]["n_jobs"] == -1:
-                params["job_kwargs"]["n_jobs"] = os.cpu_count()
-    
```

Comment:
Th is not a good idea to move this to `split_job_kwargs()` because this split function is not always call befoire this.

Reply:
oups sorry this is call after.